ios/App/App/plugins/AbsAudioPlayer.swift

```diff
@@ -95,13 +99,69 @@ public class AbsAudioPlayer: CAPPlugin {
     }
     
     @objc func sendMetadata() {
-        self.notifyListeners("onPlayingUpdate", data: [ "value": !PlayerHandler.paused() ])
+        self.notifyListeners("onPlayingUpdate", data: [ "value": !PlayerHandler.paused ])
         self.notifyListeners("onMetadata", data: PlayerHandler.getMetdata())
     }
     @objc func sendPlaybackClosedEvent() {
         self.notifyListeners("onPlaybackClosed", data: [ "value": true ])
     }
     
+    @objc func decreaseSleepTime(_ call: CAPPluginCall) {
+        guard let timeString = call.getString("time") else { return call.resolve([ "success": false ]) }
+        guard let time = Int(timeString) else { return call.resolve([ "success": false ]) }
+        guard let currentSleepTime = PlayerHandler.remainingSleepTime else { return call.resolve([ "success": false ]) }
+        
+        PlayerHandler.remainingSleepTime = currentSleepTime - time
+        call.resolve()
+    }
+    @objc func increaseSleepTime(_ call: CAPPluginCall) {
+        guard let timeString = call.getString("time") else { return call.resolve([ "success": false ]) }
+        guard let time = Int(timeString) else { return call.resolve([ "success": false ]) }
+        guard let currentSleepTime = PlayerHandler.remainingSleepTime else { return call.resolve([ "success": false ]) }
+        
+        PlayerHandler.remainingSleepTime = currentSleepTime + time
```

Comment:
Time is coming in milliseconds. We need to change to
`PlayerHandler.remainingSleepTime = currentSleepTime + (time / 1000)`